datadog-agent.yaml

```diff
@@ -270,10 +414,15 @@ test:
     contents:
       packages:
         - datadog-agent-fakeintake
+        - datadog-agent-core-integrations
+    environment:
+      PYTHONPATH: "/usr/share/datadog-agent/lib/python3.11/site-packages"
```

Comment:
did the previous use of PATH not work? i think i kind of prefer it to this.

i'm not fussed about it though.

we have lots of this stuff to "best practicify"

Reply:
the current package didn't build the core-integrations correctly so it was never loading to begin with.

I kept things namespaced in `/usr/share` because directly installing was conflicting with some system build libraries. I imagine I was just doing things wrong, but iiuc the end result is mostly the same. if that's correct I'll just leave it as is for now until the best practices are more solidified 🙏